src/components/BookCard.js

```diff
@@ -78,24 +78,23 @@ const Card = styled.section`
 `
 const CoverWrapper = styled.span`
   display: grid;
-  align-content: center;
+  align-content: start;
   justify-content: center;
-  padding: 10px;
-`
-const Author = styled.h2`
-  font-weight: normal;
-  font-size: 90%;
+  padding: 15px 10px;
 `
 const Title = styled.h2`
   font-weight: bold;
   font-size: 100%;
   padding-right: 40px;
 `
+const Author = styled.h2`
```

Comment:
Are `Title` and `Author` hierarchically the same? As in are they both equally important? If not, then `Author` should be a `h3`-element. Just something to think about :)